src/cmd/config/mock_test.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/rest"
+)
+
+func TestMockConfigProvider(t *testing.T) {
```

Comment:
Why test mocks?

Reply:
I used TDD for the most parts, so the tests came first